### PR TITLE
[ADD] OrderItem model

### PR DESCRIPTION
**Pull request by DanielHernandezO**
added OrderItem model with migrations, setters and getters

### CODE REVIEW

File: app/Models/OrderItem.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    {
+        return $this->attributes['id'];
+    }
+
+    public function getQuantity(): int
+    {
+        return $this->attributes['quantity'];
+    }
+
+    public function setTotalAmount(int $quantity): void
```
Comment (Jguerra47) [inline app/Models/OrderItem.php:26]: rename the setter
```suggestion
    public function setQuantity(int $quantity): void
```

File: app/Models/OrderItem.php [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class OrderItem extends Model
+{
+    /**
+     * ORDERITEM ATTRIBUTES
+     * $this->attributes['id'] - int - contains the review primary key (id)
```
Comment (Jguerra47) [inline app/Models/OrderItem.php:11]: Review PK?

Comment (DanielHernandezO) [reply app/Models/OrderItem.php]: As you can see it is OrderItem

**Review by Jguerra47 — APPROVED** [review]
LGTM 🙌

**Pull request merged**